core/client/db/src/light.rs

```diff
@@ -871,7 +870,7 @@ pub(crate) mod tests {
 	fn authorities_are_cached() {
 		let db = LightStorage::new_test();
 
-		fn run_checks(db: &LightStorage<Block>, max: u64, checks: &[(u64, Option<Vec<AuthorityIdFor<Block>>>)]) {
+		fn run_checks(db: &LightStorage<Block>, max: u64, checks: &[(u64, Option<Vec<AuthorityId>>)]) {
```

Comment:
cc @svyatonik are there plans to replace this with the generic runtime cache system?

Reply:
I'm not sure what do you mean by generic runtime cache system, but the it is generic enough to hold anything that is `Encode + Decode`

Reply:
What I mean is that this function references `AuthorityId`, which we don't want. What steps need to be taken to generalize that?

Reply:
It's just the test :) It uses `AuthorityId` by historical reasons. I think we could easily switch it to use `u64` instead.